test/src/api/commands/client/testWaitUntil.js

```diff
@@ -252,5 +252,38 @@ describe('.waitUntil()', function () {
 
       this.client.start(done);
     });
+
+    it('client.waitUntil() function failure with custom waitForConditionPollInterval', function (done) {
+      let tries = 0;
+      let startTime = new Date().valueOf();
+      let timeDiff;
+      const maxTimeout = 100;
+      const client = this.client.api;
+      let result;
+
+      client.globals.waitForConditionPollInterval = 11;
+
+      this.client.api.waitUntil(function () {
+        assert.deepStrictEqual(this.options, client.options);
+        tries++;
+
+        return false;
+      }, maxTimeout, '200', 'custom error message', function(response) {
```

Comment:
Shouln't this be an integer?

Reply:
should be `null` actually for this test.